src/ol_infrastructure/applications/keycloak/keycloak_server_policy.hcl

```diff
@@ -0,0 +1,17 @@
+path "secret-operations/*" {
```

Comment:
This seems a bit broad? Maybe scope it specifically to a path that holds the relevant Keycloak secrets?

Reply:
Yeah I think this was laziness on my part and not knowing what I needed from secret-operations/ yet. I think at the moment we just need the wildcard cert and I did make a secret-keycloak/* mount explicitly for this.